### ISSUE TITLE
Integrate 3D rendering

### ISSUE THREAD

**Issue by hrj** [librecaptcha/lc-core#5]
Possible Renderers:
* Cycles from blender
* [Sunflow](http://sunflow.sourceforge.net/index.php), ray tracer in pure Java. (Updated fork [here](https://github.com/monkstone/sunflow)).

**Comment by blackmagiccoffee** [comment]
I have not looked too closely into this but is there a reason LWJGL is off the table here?  Sunflow seems to be dead, that updated fork is 6 years old.

**Comment by hrj** [comment]
@blackmagiccoffee LWJGL, or any other OpenGL / Vulkan wrapper, would be too low level. Using them would require setting up vertex buffers and triangle meshes, etc.

I was hoping for a library that provides higher level of abstraction, such as "ray trace this 3D mesh with this camera and these lights". Preferably with a CPU rendering fallback, since GPU servers need a lot of moolah 💸 .

Maybe, game libraries such as libGDX and jMonkeyEngine would do.

**Comment by blackmagiccoffee** [comment]
@hrj Fair enough.  Its been at least 5-7 years since I have used either of them but I feel of those two libGDX would be easier to get going.
